### PR TITLE
Update README to say that using `requirement()` is optional

### PR DESCRIPTION
**Pull request by person142**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
Issue Number: https://github.com/bazelbuild/rules_python/issues/414

Currently the documentation recommends only using `requirement()` to access the `py_library` targets generated by `pip_install` and `pip_parse`.

## What is the new behavior?
Update the documentation to specify that using `requirement()` is optional, and that accessing the labels directly is ok.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by alexeagle — APPROVED** [review]
Awesome thank you for picking up that TODO :)

File: README.md [hunk 1]
```diff
@@ -169,12 +168,37 @@ py_library(
...
+```
+
+Here `name` is the `name` attribute that was passed to `pip_parse` and
+`package` is the pip package name with characters that are illegal in
+Bazel label names (e.g. `-`, `.`) replaced with `_`. If you need to
+update `name` from "old" to "new", then you can run the following
+buildozer command:
+
+```
+buildozer 'substitute deps @old_([^/]+)//:pkg @new_${1}//:pkg' //...
```
Comment (aignas) [inline README.md:192]: This does not work as the target list needs to be specified as `//...:*` for all of the targets.

Comment (hrfuller) [reply README.md:192]: good point @aignas

Comment (person142) [reply README.md:192]: Derp, I ran this locally to test but copy pasted wrong apparently...

**Pull request merged**